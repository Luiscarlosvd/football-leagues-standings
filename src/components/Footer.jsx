const Footer = () => {
  return (
    <div className="bg-blue-900 p-7 text-white">
      <p className="text-lato-400">Created by Luiscarlosvd under CC License</p>
    </div>
  )
}

export default Footer
