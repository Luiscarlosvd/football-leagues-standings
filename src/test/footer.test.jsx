import { render } from '@testing-library/react';
import Footer from '../components/Footer';
import { describe, it, expect } from 'vitest';
import { BrowserRouter } from 'react-router-dom';

describe('Footer', () => {
  it('should render the footer correctly', () => {
    const { container } = render(
        <BrowserRouter>
            <Footer />
        </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});