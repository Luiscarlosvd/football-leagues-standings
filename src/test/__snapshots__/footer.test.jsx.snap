// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > should render the footer correctly 1`] = `
<div>
  <div
    class="bg-blue-900 p-7 text-white"
  >
    <p
      class="text-lato-400"
    >
      Created by Luiscarlosvd under CC License
    </p>
  </div>
</div>
`;
